Aptoide/aptoide-client-v8
# [feature/MOB-238] Installed With Aptoide

**What does this PR do?**

   Implements the new "Installed with Aptoide" feature, including the AB Test. Note that this implementation only relies on the database for now (i.e. it's not taking into consideration the new split installation method, where we mark that an app was installed with Aptoide). However, it is easily added if needed in the future.

**Database changed?**

   Yes

**How should this be manually tested?**

  First, check that you're lading on the "show_installed" (b) group of the AB Test (you may override the response with Charles). Then, every app that is installed or updated with Aptoide should now mark that app as "Installed with Aptoide" in Apps. This mark is only reflected in the Updates section. Also check that there's impression and conversion events sent to Rakam (impression when you open apps, conversion when you start an update).

**What are the relevant tickets?**

  [MOB-238](https://aptoide.atlassian.net/browse/MOB-238)


**Code Review Checklist**
